bedevere/close_pr.py

```diff
@@ -0,0 +1,27 @@
+"""Automatically close PR that tries to merge maintenance branch into master."""
+import re
+
+import gidgethub.routing
+
+
+PYTHON_MAINT_BRANCH_RE = re.compile(r'^python:\d+.\d+$')
+
+router = gidgethub.routing.Router()
+
+@router.register("pull_request", action="opened")
+@router.register("pull_request", action="synchronize")
+async def close_invalid_pr(event, gh, *args, **kwargs):
+    """Close the invalid PR.
+    PR is considered invalid if:
+    base_label is 'python:master', and
```

Comment:
Might as well make this a list, i.e.:
```restructuredtext
* base_label is 'python:master'
* head_label is 'python:<some maintenance branch>'
```

Otherwise put a period at the end of the sentence.